cmd/run.go

```diff
@@ -74,6 +74,14 @@ func createEmailPool(config config.Provider) (*email.SMTPClient, error) {
 	}
 	user := config.GetString("email.username")
 	pass := config.GetString("email.password")
+	passFile := config.GetString("email.password_file")

```

Comment:
I would warn if there is pass && passFile are defined.

Reply:
With a warning log or would you stop the execution?

Reply:
Warning is fine IMHO

Reply:
done